messaging/src/main/java/org/axonframework/eventhandling/AbstractEventBus.java

```diff
@@ -283,6 +305,18 @@ public Builder messageMonitor(@Nonnull MessageMonitor<? super EventMessage<?>> m
             return this;
         }
 
+        /**
+         * Sets the {@link SpanFactory} implementation to use for providing tracing capabilities.
```

Comment:
Note that the `builder` methods throughout also define the default values. Hence, I'd expect all the infra components where you have added this default field to have a change in the `builder`'s JavaDoc too.